src/azure-cli/azure/cli/command_modules/backup/_help.py

```diff
@@ -26,6 +26,22 @@
     crafted: true
 """
 
+helps['backup container register'] = """
+type: command
+short-summary: This command allows Azure Backup to register a Resource to the given Recovery Services Vault.
+examples:
+  - name: This command allows Azure Backup to convert the 'Resource' to a 'Backup Container' which is then registered to the given Recovery services vault. The Azure Backup service can then discover workloads of the given workload type within this container to be protected later.
+    text: az backup container register --resource-group MyResourceGroup --vault-name MyVault --resource-id MyResourceId --workload-type MSSQL --backup-management-type AzureWorkload --resource-id MyResourceID
+"""
+
+helps['backup container re-register'] = """
+type: command
+short-summary: Resets the registration details for a given container.
+examples:
+  - name: Resets the registration details for a given container. To be used only in error scenarios as specified here (https://docs.microsoft.com/azure/backup/backup-sql-server-azure-troubleshoot#re-registration-failures). Understanding the failure symptoms and causes before attempting re-registration.
```

Comment:
Resets -> Reset

Reply:
Agreed

Reply:
@sambitratha : please change the last sentence to "Understand the failure symptoms and causes before attempting re-registration"